src/components/dialogmoteikkeaktuell/DialogmoteikkeaktuellSkjema.tsx

```diff
@@ -0,0 +1,139 @@
+import React from "react";
+import { Link, Navigate } from "react-router-dom";
+import { moteoversiktRoutePath } from "@/routers/AppRouter";
+import { useDialogmotekandidat } from "@/data/dialogmotekandidat/dialogmotekandidatQueryHooks";
+import {
+  CreateIkkeAktuellDTO,
+  IkkeAktuellArsak,
+} from "@/data/dialogmotekandidat/types/dialogmoteikkeaktuellTypes";
+import { useValgtPersonident } from "@/hooks/useValgtBruker";
+import { useSettDialogmoteikkeaktuell } from "@/data/dialogmotekandidat/useSettDialogmoteikkeaktuell";
+import { SkjemaInnsendingFeil } from "@/components/SkjemaInnsendingFeil";
+import { SubmitHandler, useForm } from "react-hook-form";
+import {
+  Alert,
+  Box,
+  Button,
+  Radio,
+  RadioGroup,
+  Textarea,
+} from "@navikt/ds-react";
+
+export const texts = {
+  noBrev: "Det blir ikke sendt ut brev.",
+  infoKandidatlist: `Når du setter ikke aktuell vil arbeidstakeren bli fjernet fra kandidatlisten. Dersom du på et senere tidspunkt vurderer at det likevel er nødvendig med et dialogmøte, kan du kalle inn til dialogmøte ved å søke deg frem til denne arbeidstakeren.`,
+  arsakLegend: "Årsak til ikke aktuell (obligatorisk)",
+  arsakErrorMessage: "Vennligst angi årsak.",
+  beskrivelseLabel: "Beskrivelse (valgfri)",
+  send: "Sett ikke aktuell",
```

Comment:
Hva med `Lagre`?

Reply:
Dette er jo *sterkt* inspirert av løsningen for unntak 😉

Der har knappen teksten "Sett unntak", så da virker "Sett ikke aktuell" bra her synes jeg.